packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -638,6 +639,11 @@ function transform(codeGen: CodeGen): t.Expression {
                 )
             );
         }
+
+        if (isExternalComponent(element)) {
+            data.push(t.property(t.identifier('external'), t.literal(true)));
```

Comment:
nitpick (non-blocking): should 'external' be a constant somewhere else? 
Also maybe adding a comment about what this is doing so it is easier to understand in the future?

Reply:
I think it's fine to leave this as `'external'` for now, at least until we start using it in multiple places.

I'm not sure what kind of comment would be helpful here since the logic seems straightforward, do you have a suggestion?